overworld/Player.gd

```diff
@@ -14,13 +14,24 @@ var Raycasts = {
 	"up": "RayCastUp",
 	"down": "RayCastDown"
 }
-var previousDir
+
+var stepsTaken = 0
+var stepsToTriggerCombat = 20
+var encounterStep = 20
+
 signal collided
 func _ready():
-	overworld = get_parent()
+	overworld = get_node("../..")
```

Comment:
I think this is the only relative path I've seen you create. From a consistency point of view, does it need to be done this way, or can you do this with an absolute path?

Reply:
fixed